recipes-extended/xen/files/0001-tools-xl-Add-stubdomain_cmdline-option-to-xl.cfg.patch

```diff
@@ -0,0 +1,87 @@
+From ce233b10319dfb77ab5e0102202190a5911ae8f3 Mon Sep 17 00:00:00 2001
```

Comment:
I assume this is also something that's going to be in 4.17 on release?

Reply:
It was needed for 4.14 and goes away for 4.16.